Delegate/SimpleTypeError.pm

```diff
@@ -433,6 +432,10 @@ sub get_delegate_info {
         undef,
         'File names may not contain a leading dot.',
     ],
+    LOCKED_OUT => [
+        83,
+        # No description as this error is not shown to the user.
```

Comment:
Description needs to come from facade because it should include a link to contact support. The default description won't be shown therefore.